templates/flavors/kubeadm/full/kustomization.yaml

```diff
@@ -41,6 +41,47 @@ patches:
               - label: etcd_data
                 filesystem: ext4
                 device: /dev/sdc
+            partitions:
+              - device: /dev/sdc
+                layout: true
           mounts:
-            - - LABEL=etcd_data
```

Comment:
Not sure about this change, I was following https://cluster-api.sigs.k8s.io/tasks/bootstrap/kubeadm-bootstrap/:
> KubeadmConfig.Mounts specifies a list of mount points to be setup.
> ```
> mounts:
> - - LABEL=etcd_disk
>   - /var/lib/etcddisk
>```

Reply:
Without this change, we noticed that the filesystem was not being mounted and the `var/lib/etcd_data` was on the root disk itself